docroot/jpipjs/astronomy.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
What is this jpipjs page? It's not accessible on the production site.

Reply:
That's from another website - it's not ours and should not appear in our repo.

Reply:
Ok, will remove it. That will shrink this a lot.

Reply:
Yes, the entire jpipjs directory should not be part of our rep0.